tests/test_rpi_models.py

```diff
@@ -25,7 +25,7 @@ def test_model_A(self):
         self.assertIn(getrpi('0008'), modela)
         self.assertIn(getrpi('0009'), modela)
 
-    def test_model_A_plus(self):
+    def test_module_a_plus(self):
```

Comment:
I just removed a duplicate of the same test. However, I didn't want to do more changes there.